frontend/app/src/store/blockchain/index.ts

```diff
@@ -113,10 +114,11 @@ export const useBlockchainStore = defineStore('blockchain', () => {
       return;
     }
 
+    const addr = await Promise.all(
+      payload.map(data => addAccount(blockchain, data))
+    );
```

Comment:
Any specific reason this is now out of the try catch?

Reply:
So what I bring out from the catch is the one that register the address.
Sometime this address is typo for example (yabir.ett). If we put this inside this try, it will use the catch below which is open the dialog. But if I bring it out, it will trigger the catch from `<AccountForm>` instead, that show the error exactly below the address field. This is behaviour I wanted. 

Hdyt, how to make it better?

Reply:
This works when you add one address well, but we also have the multi-address addition. What happens if only one out of five addresses is wrong?